Proposal: website-deploy

victorb opened the pull request:
Here is some more thoughts around website-deploy, a static website build service. Thoughts written here are unfiltered, might contain allergens and other goodies. 

If you have any feedback, please leave them in a comment here. If something looks weird in the proposal, please comment on the line directly.

Rendered preview: https://github.com/open-services/open-services/blob/67ecf170835d100ec8e40b9784b8f546b6d3e8b3/services/1-evaluating/website-deploy.md

CC @agentofuser as I know you're interested in this :)

victorb committed:
Add thoughts about website-deploy

License: MIT
Signed-off-by: Victor Bjelkholm <git@victor.earth>

agentofuser commented:
Yes! Thanks for the heads up :) I'm a little busy with client work right now, but I'll take a look first chance I get.

victorb committed:
Add some more notes

Specifically about web interface uploads, reusable setup steps, dynamic
princing, handling incoming deletion requests and analytics

License: MIT
Signed-off-by: Victor Bjelkholm <git@victor.earth>
